2025-09-10 23:33:41,235 DEBUG    root            Loaded Command Group: ['gcloud', 'storage']
2025-09-10 23:33:41,235 DEBUG    root            Loaded Command Group: ['gcloud', 'storage', 'buckets']
2025-09-10 23:33:41,360 DEBUG    root            Loaded Command Group: ['gcloud', 'storage', 'buckets', 'list']
2025-09-10 23:33:41,361 DEBUG    root            Running [gcloud.storage.buckets.list] with arguments: [--limit: "10"]
2025-09-10 23:33:41,361 DEBUG    root            Chosen display Format:default
2025-09-10 23:33:41,362 INFO     root            Display format: "default"
2025-09-10 23:33:41,369 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): storage.googleapis.com:443
2025-09-10 23:33:42,038 DEBUG    urllib3.connectionpool https://storage.googleapis.com:443 "GET /storage/v1/b?alt=json&maxResults=1000&project=lionsofzion-dev&projection=full HTTP/11" 403 530
2025-09-10 23:33:42,039 DEBUG    root            (gcloud.storage.buckets.list) HTTPError 403: admin@lionsofzion-official.org does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist). This command is authenticated as admin@lionsofzion-official.org which is the active account specified by the [core/account] property.
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/api_lib/storage/gcs_json/client.py", line 602, in list_buckets
    for bucket in bucket_iter:
                  ^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/list_pager.py", line 120, in YieldFromList
    response = getattr(service, method)(request,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/storage/v1/storage_v1_client.py", line 565, in List
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 747, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 753, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 612, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpForbiddenError: HttpError accessing <https://storage.googleapis.com/storage/v1/b?alt=json&maxResults=1000&project=lionsofzion-dev&projection=full>: response: <{'content-type': 'application/json; charset=UTF-8', 'x-guploader-uploadid': 'ABgVH8-GwO4OvkQal9b362Tyh64grWmSvxln_K-_Ma8tZ3PQTnBDD8sFtGXMBbX_7Aqm9plhoS-Cq7o', 'date': 'Wed, 10 Sep 2025 20:33:42 GMT', 'vary': 'Origin, X-Origin', 'cache-control': 'no-cache, no-store, max-age=0, must-revalidate', 'expires': 'Mon, 01 Jan 1990 00:00:00 GMT', 'pragma': 'no-cache', 'content-length': '530', 'server': 'UploadServer', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'status': 403}>, content <{
  "error": {
    "code": 403,
    "message": "admin@lionsofzion-official.org does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist).",
    "errors": [
      {
        "message": "admin@lionsofzion-official.org does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist).",
        "domain": "global",
        "reason": "forbidden"
      }
    ]
  }
}
>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 949, in Run
    ).Display()
      ^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/calliope/display.py", line 507, in Display
    self._printer.Print(self._resources)
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/core/resource/resource_printer_base.py", line 301, in Print
    for resource in resources:
                    ^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/core/util/peek_iterable.py", line 211, in __next__
    item = self._NextItem()
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/core/util/peek_iterable.py", line 183, in _NextItem
    return next(self._iterable)
           ^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/surface/storage/buckets/list.py", line 84, in Run
    for bucket in wildcard_iterator.get_wildcard_iterator(
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/command_lib/storage/wildcard_iterator.py", line 432, in __iter__
    for bucket_resource in self._client.list_buckets(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/api_lib/storage/gcs_json/client.py", line 605, in list_buckets
    core_exceptions.reraise(
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/core/exceptions.py", line 149, in reraise
    six.reraise(type(exc_value), exc_value, tb)
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/six/__init__.py", line 718, in reraise
    raise value.with_traceback(tb)
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/api_lib/storage/gcs_json/client.py", line 602, in list_buckets
    for bucket in bucket_iter:
                  ^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/list_pager.py", line 120, in YieldFromList
    response = getattr(service, method)(request,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/storage/v1/storage_v1_client.py", line 565, in List
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 747, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 753, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Caskroom/gcloud-cli/538.0.0/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 612, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
googlecloudsdk.api_lib.storage.errors.GcsApiError: HTTPError 403: admin@lionsofzion-official.org does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist). This command is authenticated as admin@lionsofzion-official.org which is the active account specified by the [core/account] property.
2025-09-10 23:33:42,046 ERROR    root            (gcloud.storage.buckets.list) HTTPError 403: admin@lionsofzion-official.org does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist). This command is authenticated as admin@lionsofzion-official.org which is the active account specified by the [core/account] property.
